util/crs-rules-check/rules-check.py

```diff
@@ -339,4 +382,17 @@ def msg(msg):
                 errmsgf(e)
             errmsg(d.strip("\n"))
 
+        ### check `ctl:auditLogParts=+E` right place in chained rules
+        c.check_ctl_audit_log()
+        if len(c.auditlogparts) == 0:
+            msg(" 'ctl:auditLogParts' actions are in right place.")
```

Comment:
Maybe `'ctl:auditLogParts' is in the last rule of the chain` ?

Reply:
Same as above - I do not know, what would be the better. Note, that I asked @RedXanadu for help to make messages :).

Reply:
Sorry, an example here too:

```
Config file: ../../rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf
 Parsing ok.
 Ignore case check ok.
 Action order check ok.
 Indentation check ok.
 'ctl:auditLogParts' actions are in right place.
```